ccm_web/package.json

```
@@ -5,7 +5,8 @@
   "scripts": {
     "dev:server": "nodemon --inspect=0.0.0.0:9229 --config nmon_config.json server/src/main.ts",
     "dev:client": "webpack --watch --config webpack/webpack.dev.ts",
-    "dev": "npm run dev:server & npm run dev:client",
+    "predev": "wait-port ccm_db:3306",
+    "dev": "node -r ts-node/register server/migrator up && npm run dev:server & npm run dev:client",
```

Isn't there supposed to be another "`&`" in that line?

```suggestion
    "dev": "node -r ts-node/register server/migrator up && npm run dev:server && npm run dev:client",
```

---

I had to look this up, since I'm not a Bash/Shell whiz. We do want it to be single `&` in the second case since we want `dev:server` (`nodemon` process that runs the server and watches for changes) and `dev:client` (`webpack` process that builds the client and watches for changes) to run simultaneously, not in sequence. The `&&` before `migrator` seems correct, since the database should be all set up before the server starts.